win32_deps_build.sh

```diff
@@ -70,6 +70,7 @@ case "$OS" in
             libtool \
             ninja-build \
             zip \
+            bzip2 \
```

Comment:
With `bzip2` on the list, do we still need `zip`?

I think both are a bit bogus because this script is also dealing with `.xz` which is more likely to be missing (at least historically), but `xz-utils` isn't on the list.

Reply:
``win32_build.sh`` creates a ZIP archive of the resulting binaries that can easily be moved around and used on Windows: https://github.com/ceph/ceph/blob/6c57f6b2c2b0297e2b394e8c66666b301eb4d6cd/win32_build.sh#L167.

If I remember correctly, some Ubuntu container images did not have ``zip`` so I've added it to the dependency list. Out of caution, I've added ``bzip2`` as well.

Reply:
Sorry, I confused `zip` with `gzip` -- I haven't dealt with Windows in a long while...  Should `xz-utils` be on the list?

Reply:
We can, although we haven't had issues with it so far. I'll need to check the package name for each of the supported distros (rhel, ubuntu and suse).

Reply:
You're right, xz-utils (or xz on rhel/suse) should also be on that list. Thanks!